content/help/ops/traffic-management/deploy-guidelines/index.md

```diff
@@ -232,3 +232,49 @@ $ kubectl scale --replicas=0 deploy/istio-citadel -n istio-system
 {{< /text >}}
 
 This should stop Istio from restarting Envoy and disconnecting TCP connections.
+
+## Configure a multiple host gateway with SSL
+
+To configure the ingress gateway with TLS correctly for you to deploy multiple
+hosts, you must configure a unique port name with the following command:
```

Comment:
As I said in #2321, this doesn't describe the general problem. I suggest something like the following:

## Configuring multiple TLS hosts in a gateway

If you configure multiple gateway servers, either in a single `Gateway`, or spread across more than one that use the same `selector` labels, with the same port number and protocol HTTPS, then you must ensure that the corresponding port names are unique. For example: